src/app/components/message/Reply.css.ts

```diff
@@ -5,6 +5,23 @@ export const ReplyBend = style({
   flexShrink: 0,
 });
 
+export const ThreadIndicator = style({
+  opacity: config.opacity.P300,
+  gap: '0.125rem',
+  cursor: 'pointer',
+
+  selectors: {
+    ':hover&': {
+      opacity: config.opacity.P500,
+    },
+  },
+});
+
+export const ThreadIndicatorIcon = style({
+  width: '0.875rem',
+  height: '0.875rem',
```

Comment:
I tried using `size` and `gap` props from `folds`, but the smallest size was still too big for this element.

Reply:
Please use `toRem` function from folds to write `rem` values in `px` as `rem` unit are hard to coordinate with designs.